File: CS/PivotGridCustomization.Module/TestObject.cs
using System;
using System.Collections.Generic;
using System.Text;
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Xpo;

namespace PivotGridCustomization.Module {
    [DefaultClassOptions]
    [VisibleInReports]
    public class TestObject : BaseObject {
        public TestObject(Session session) : base(session) { }

        string name;
        private int value;
        private int anotherValue;

        public string Name {
            get { return name; }
            set { SetPropertyValue("Name", ref name, value); }
        }
        public int Value {
            get { return value; }
            set { SetPropertyValue("Value", ref this.value, value); }
        }
        public int AnotherValue {
            get { return anotherValue; }
            set { SetPropertyValue("AnotherValue", ref this.anotherValue, value); }
        }
    }
}

File: VB/PivotGridCustomization.Module/TestObject.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Xpo

Namespace PivotGridCustomization.Module
	<DefaultClassOptions, VisibleInReports> _
	Public Class TestObject
		Inherits BaseObject
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub

		Private name_Renamed As String
		Private value_Renamed As Integer
		Private anotherValue_Renamed As Integer

		Public Property Name() As String
			Get
				Return name_Renamed
			End Get
			Set(ByVal value As String)
				SetPropertyValue("Name", name_Renamed, value)
			End Set
		End Property
		Public Property Value() As Integer
			Get
				Return value_Renamed
			End Get
			Set(ByVal value As Integer)
				SetPropertyValue("Value", Me.value_Renamed, value)
			End Set
		End Property
		Public Property AnotherValue() As Integer
			Get
				Return anotherValue_Renamed
			End Get
			Set(ByVal value As Integer)
				SetPropertyValue("AnotherValue", Me.anotherValue_Renamed, value)
			End Set
		End Property
	End Class
End Namespace
